build/NetFrameworkRelease.targets

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+	<NoWarn>RS0016</NoWarn>
```

Comment:
This warning is about not tracking publicly exposed methods and classes (an API) using 2 files--PublicAPI.Shipped.txt and PublicAPI.Unshipped.txt. As an app, we don't expose an API surface, so there's no point in adding the overhead of tracking one.